src/style.css

```diff
@@ -1,18 +1,26 @@
+body {
+  background-color: #8ebfe7;
+}
+
 header {
-  background-color: #000;
```

Comment:
- Nice Job 🥇 🥇🥇

![Capture](https://user-images.githubusercontent.com/63972670/202225991-76e20110-7e8e-4bd8-b708-cac93716580a.JPG)

Reply:
do i need to change anything on that? @OLIPLICHE

Reply:
-Meet me on slack